delta_shooter/game_mechanics.py

```diff
@@ -106,7 +106,7 @@ def choose_move_randomly(state: np.ndarray) -> int:
     return np.random.randint(6)
 
 
-class ShooterEnv(gym.Env):
+class ShooterEnv:
```

Comment:
Shall we remove `gym` from imports and `requirements.txt`?

Reply:
yass